contrib/automation_tests/orbit_instrument_function.py

```diff
@@ -14,52 +20,20 @@
  - instrument a function
  - take a capture and verify the hooked function is recorded
 """
+import orbit_testing
 import logging
 import time
 from absl import app
 import pywinauto
 from pywinauto.application import Application
 
-
-def WaitForOrbit():
-  while True:
-    try:
-      Application(backend='uia').connect(title_re='orbitprofiler')
-      break
-    except pywinauto.findwindows.ElementNotFoundError:
-      pass
-
-
 def main(argv):
-  WaitForOrbit()
+  orbit_testing.WaitForOrbit()
   application = Application(backend='uia').connect(title_re='orbitprofiler')
+  orbit_testing.ConnectToGamelet(application)
+  orbit_testing.SelectProcess(application, 'hello_')
 
-  if application.is_process_running():
```

Comment:
Not sure if you wanted to keep this if / else here. It was not present in the preset test.

Reply:
Please remove ... this is left over from some debugging I guess. This adds nothing here since 
`application = Application(backend='uia').connect(title_re='orbitprofiler')
`above fails if Orbit is not running.